tests/resources/VCH-Util.robot

```diff
@@ -35,7 +35,7 @@ Set Test Environment Variables
     @{URLs}=  Split String  %{TEST_URL_ARRAY}
     #${len}=  Get Length  ${URLs}
     # hardcode the TEST_URL_Array number as 10 since the other 6 HAAS servers are taken for other usage
-    ${len}=  Set Variable if  %{DRONE_BUILD_NUMBER}  4  1
+    ${len}=  Set Variable if  %{DRONE_BUILD_NUMBER}  5  1
```

Comment:
Can you give more description about the purpose in the commit message?

Reply:
wending is adding another idle server for VIC CI testing.